codebuild/fuzz_codebuild.config

```diff
@@ -121,3 +121,8 @@ start_time: 05:00
 build_job_name: s2nFuzzScheduled
 input: {"environmentVariablesOverride": [{"name": "FUZZ_TESTS","value": "s2n_stuffer_pem_fuzz_test"}]}
 
+[CloudWatchEvent:s2n_deserialize_resumption_state_test]
+start_time: 05:00
```

Comment:
super cool to see more fuzzing!

thoughts... we should stagger this time to avoid launching all tests at once if we start to add more fuzzing.